go.mod

```diff
@@ -1,15 +1,15 @@
 module github.com/terraform-providers/terraform-provider-dnsimple
 
 require (
-	github.com/dnsimple/dnsimple-go v1.2.0
+	github.com/dnsimple/dnsimple-go v1.2.2-0.20230824051107-a0bab607f157
```

Comment:
Should this be `v1.3.0`?

Reply:
Changed in [5e239dd](https://github.com/dnsimple/terraform-provider-dnsimple/pull/142/commits/5e239dda06b9d0bed9c1d101cbce2a7432446769)